internal/pkg/cli/app_init.go

```diff
@@ -165,10 +169,11 @@ func (o *initAppOpts) Execute() error {
 	}
 	o.prog.Start(fmt.Sprintf(fmtAppInitStart, color.HighlightUserInput(o.name)))
 	err = o.cfn.DeployApp(&deploy.CreateAppInput{
-		Name:           o.name,
-		AccountID:      caller.Account,
-		DomainName:     o.domainName,
-		AdditionalTags: o.resourceTags,
+		Name:             o.name,
+		AccountID:        caller.Account,
+		DomainName:       o.domainName,
+		DomainHostedZone: o.domainHostedZoneID,
```

Comment:
nit: just a thought, i'm wondering if we should unify the field names? So that it's `DomainHostedZoneID: o.domainHostedZoneID` or `DomainHostedZone: o.domainHostedZone`